src/main/java/seedu/duke/command/FindCommand.java

```diff
@@ -75,11 +69,13 @@ public void execute(TransactionList transactions, Ui ui, Storage storage) {
         try {
             // Checks the format of find to ensure that it contains keywords used in the search expression
             checkFindFormat(keywords);
+            assert !keywords.isBlank();

```

Comment:
Here, we are asserting whether keywords is not blank. Should there be an error message should this assertion fails?

Reply:
Yup an error message will be generated by the assertion automatically ^^